code/game/objects/items/ego_weapons/non_abnormality/cinq.dm

```diff
@@ -28,7 +28,7 @@
 	if(!ready)
 		return
 	ready = FALSE
-	to_chat(user, "<span class='userdanger'>Allons-y!</span>")
+	to_chat(user, "<span class='danger'>Allons-y!</span>")
```

Comment:
should probably be changed into a proper span class whilst you are there
```suggestion
	to_chat(user, span_danger("Allons-y!"))
```

Reply:
Very true